README.md

```diff
@@ -112,24 +112,24 @@ source ./setup_shell.sh
 ### 3. Setting up vendors
 
 RAI is vendor-agnostic. Use the configuration in [config.toml](./config.toml) to set up your vendor of choice for RAI modules.
+Vendor choices for RAI are summarized in [Vendors Overview](docs/vendors_overview.md).
 
-If you do not have a vendor's key, follow the instructions below:
+For the core AI service, follow these instructions for a locally hosted Ollama [link](https://ollama.com/download).
```

Comment:
Can we please add information that we advise to use paid models as they are MUCH better with RAI at the moment? I did some tests, @boczekbartek did too, and as of now RAI with llama does not represent the RAI's quality at all...